Description: use parent.pm instead of base.pm
 The later has issues in the aftermath of the .-in-@INC security issue.
Origin: vendor
Bug: https://rt.cpan.org/Ticket/Display.html?id=116589
Bug-Debian: https://bugs.debian.org/832862
Forwarded: https://rt.cpan.org/Ticket/Display.html?id=116589
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2016-08-18

--- a/t/Graph/Writer/DSM.t
+++ b/t/Graph/Writer/DSM.t
@@ -1,5 +1,5 @@
 package Test::Graph::Writer::DSM;
-use base qw(t::Graph::Writer::DSM::Test);
+use parent qw(t::Graph::Writer::DSM::Test);
 use Test::Most 'no_plan';
 use Test::File;
 use Graph;
--- a/t/Graph/Writer/DSM/HTML.t
+++ b/t/Graph/Writer/DSM/HTML.t
@@ -1,5 +1,5 @@
 package Test::Graph::Writer::DSM::HTML;
-use base qw(t::Graph::Writer::DSM::Test);
+use parent qw(t::Graph::Writer::DSM::Test);
 use Test::Most 'no_plan';
 use Test::File;
 use Graph;
